### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by der**
Addresses depenabot vulnerabilities.

Tested against local copies of user password store, version update appears safe and does not break existing account logins.

### CODE REVIEW

File: src/main/java/com/epimorphics/registry/RegRun.java [hunk 1]
```diff
@@ -45,8 +45,8 @@ public static void main(String[] args) throws Exception {
 
         tomcat.setBaseDir(".");
 
-        // String contextPath = "/registry";  // Use for local testing without nginx/apache front end reverse proxy
-        String contextPath = "/ldregistry";
+        String contextPath = "/registry";  // Use for local testing without nginx/apache front end reverse proxy
```
Comment (simonoakesepimorphics) [inline src/main/java/com/epimorphics/registry/RegRun.java:48]: Is this change intentional?

Comment (der) [reply src/main/java/com/epimorphics/registry/RegRun.java:48]: Yes but it could be omitted if preferred.

We typically (not not universally) deploy at the `/registry` context path and a number of canned examples I use for testing (e.g. Environment registry) follow that pattern. It seemed more useful to be able to run local manual testing directly (which is all `RegRun` is used for) without requiring a local proxy server.
